src_new/controllers/gpt.js

```diff
@@ -124,6 +124,10 @@ function getAdSlotSizesArray(divID, currentGoogleSlot) { // TDD, i/o : done
 
     if (sizeMapping !== false) {
         util.log(divID + ": responsiveSizeMapping applied: ");
+        /* istanbul ignore else */
+        if(typeof(sizeMapping[0]) == "string" || (sizeMapping[0].length == 1 && typeof(sizeMapping[0][0]) == "string")){
```

Comment:
it is not clear to me why we have added condition " (sizeMapping[0].length == 1 && typeof(sizeMapping[0][0]) == "string") "
Please elaborate the case here.
I think we are trying to handle value "fluid"

Reply:
Please add some comments in code about why we are adding a special handling

Reply:
As per gpt reference https://developers.google.com/doubleclick-gpt/reference
both 'fluid' and ['fluid'] are acceptable forms to declare a slot size as fluid. So handled scenario for the same

Reply:
ok, please mention this case in comment in code.
also, it will be good if you add isArray check, because of the following behavior in Chrome browser console

var a = 'a';
undefined

typeof a[0] 
"string"

typeof a[0][0] 
"string"

typeof a[0][0][0]
"string"

typeof a[0][0][0][1]
"undefined"

typeof a[0][0][0][0]
"string"

Reply:
Make sense 
had a isArray check in second condition before checking length == 1